plugins/modules/packaging/os/pkgng.py

```diff
@@ -22,7 +22,7 @@
 options:
     name:
         description:
-            - Name or list of names of packages to install/remove.
+            - Name or list of names of packages to install/remove. With I(name=*), I(state: latest) will operate, but I(state: present) and I(state: absent) will be noops.
```

Comment:
```suggestion
            - Name or list of names of packages to install/remove.
            - With I(name=*), I(state: latest) will operate, but I(state: present) and I(state: absent) will be noops.
```
I think with a new paragraph this will be easier to read.